specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/examples/ClustersCreate.json

```diff
@@ -0,0 +1,94 @@
+{
+  "parameters": {
+    "resourceGroupName": "oiautorest6685",
+    "clusterName": "oiautorest6685",
+    "parameters": {
+      "sku": {
+        "name": "CapacityReservation",
+        "capacity": 1000
+      },
+      "location": "australiasoutheast",
+      "tags": {
+        "tag1": "val1"
+      }
+    },
+    "api-version": "2022-10-01",
+    "subscriptionId": "00000000-0000-0000-0000-00000000000"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-00000000000/resourceGroups/oiautorest6685/providers/Microsoft.OperationalInsights/clusters/oiautorest6685/operationresults/00000000-0000-0000-0000-00000000000?api-version=2021-06-01",
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-00000000000/providers/Microsoft.OperationalInsights/locations/eastus/operationstatuses/00000000-0000-0000-0000-00000000000?api-version=2015-11-01-preview"
+      }
+    },
+    "200": {
+      "body": {
+        "properties": {
+          "keyVaultProperties": {
+            "keyVaultUri": "https://aztest2170.vault.azure.net",
+            "keyName": "aztest2170cert",
+            "keyVersion": "654ft6c4e63845cbb50fd6fg51540429"
```

Comment:
Would it make sense to just give the secrets uri, instead of all the parts like this? It seems a bit more internetty hypertexty.